procedures/MT/rotator/rotator_performance.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<p>Why is this method required? Why would you want to move the origin?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Explained in the commit: [30d19d5](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/30d19d514c64981689c466b89131d18ff78a1464). Thanks!